pkg/sources/fake/fake.go

```diff
@@ -25,10 +46,87 @@ func getCAN(id uint16) msgs.CAN {
 
 // GenFake generates fake data
 func GenFake(bus *pubsub.MessageBus) {
```

Comment:
I'm ok with this being messy for now but we should create a ticket to find a nicer way... It just looks and feels clunky. But it does the job so I won't knock it and it isn't mission critical to anything.